src/css/base/_variables.scss

```diff
@@ -39,13 +39,13 @@ $coa-xlarge-screen: 1250px;
 
 //padding / margin
 $coa-space-level1: 0.5rem;
-$coa-space-level2: 1.0rem;
+$coa-space-level2: 1rem;
```

Comment:
not considering this blocking, but there are some style tweaks on this page, not sure if we have a consistent style guide for unit numbers, color hashes, etc

Reply:
I'll have to connect with Chase to see if that's ready/available because it'll definitely have to get updated here